demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -177,9 +180,10 @@ const BACKGROUND_BLUR_ASSET_SPEC = (BACKGROUND_BLUR_ASSET_GROUP || BACKGROUND_BL
   revisionID: BACKGROUND_BLUR_REVISION_ID,
 }
 
-type VideoFilterName = 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 'Background Blur 10% CPU' | 'Background Blur 20% CPU' | 'Background Blur 30% CPU' | 'Background Blur 40% CPU' | 'Background Replacement' | 'None';
+type VideoFilterName = 'Premium Device - Blue Shift' | 'Premium Device - Red Shift' | 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 
```

Comment:
If these options are fixed, it is better to use ENUM for them

Reply:
Adjusting this to enum changes a decent amount of the configuration in the rest of the meetingV2 application. I would rather not spend too much time fixing this as Ryan is working on removing the filters from the application in their entirety.

Reply:
OK sounds good but @ryanajae please noting this when doing your task!